src/api/firebaseUtils.ts

```diff
@@ -1185,3 +1186,50 @@ export const getInterestsEvent = async (interests: string[]) => {
     }
 }
 
+const deleteUserFirestoreData = async (userId: string) => {
```

Comment:
Ooohhhhh, this is really scary. Are we sure we want to immediately delete the data? In the case of an account being compromised or a user accidentally deleting their account, this ensures no chance of recovery. Maybe we should instead have a "deleted" flag or move the data into another collection with pending deletes.